pimd/pim_register.c

```diff
@@ -346,7 +346,7 @@ pim_register_recv (struct interface *ifp,
 	  zlog_debug("%s: Sending register-Stop to %s and dropping mr. packet",
 	    __func__, "Sender");
 	/* Drop Packet Silently */
-	return 1;
+	return 0;
```

Comment:
Hello Jafar, 
Here we are dropping packet silently because there are no receivers ( no RPT) and FHR sends Register packets. If we send return 1 then it marks again errored packet counts. To stream line packet counts Tx/Rx if we are dropping silently then return 0 would be ideal.